### PR TITLE
I created simple a draft

### PR DESCRIPTION
**Pull request by hevalberknevruz**
I think, we can use like this a structure. If you have another idea, let's talk.

### CODE REVIEW

File: analyzer.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package main
+
```
Comment (hevalberknevruz) [inline analyzer.go]: Analyzers are composed of a single Tokenizer and zero or more TokenFilters. The tokenizer may be preceded by one or more CharFilters.

File: analyzer.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+type Analyzer struct {
+	CharFilters  []CharFilter
+	Tokenizer    Tokenizer
+	TokenFilters []TokenFilter
+}
```
Comment (hevalberknevruz) [inline analyzer.go]: Analyzers are composed of a single Tokenizer and zero or more TokenFilters. The tokenizer may be preceded by one or more CharFilters.

**Comment by hevalberknevruz** [comment]
Some codes was unformatted but it doesn't look like that on my computer.

File: parser.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import(
+	"log"
+	"io/ioutil"
+	"encoding/json"
+)
+
+const SAMPLE_DATA_PATH = "data.json"
+
+type Document struct{
```
Comment (manorie) [inline parser.go]: bu documan hangi user a ait gibi bir identifier lazim, yada classifier

File: parser.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+	Id string			 `json:"id"`
+	ResourceName string  `json:"resourcename"`
+	ContentType []string `json:"content_type"`
+	Content []string     `json:"content"`
+}
+
+func readFileAsByte(path string) []byte{
```
Comment (manorie) [inline parser.go]: bro burda permission hatasi vs. baska bir errorda olabilir burda olasi errorlari donebiliriz bence

File: analyzer.go (hunk 2 above)
Comment (manorie) [inline analyzer.go]: We need more pointers I think, this will destory memory as slices replicated in memory.

**Comment by manorie** [comment]
Bro, needs go fmt for code styling.

File: parser.go (hunk 3 above)
Comment (hevalberknevruz) [inline parser.go]: Onları ekleyecez zaten de hep beraber çözüm getirmemiz lazım maksat bir taslak olsun diye birşeyler yapayım dedim.

Comment (manorie) [inline parser.go]: Iyi yapmissin ben unutmayalim diye yazdim buraya :) Eline saglik.

**Pull request closed without merging**